frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -1142,7 +1143,13 @@
                     var error = response.data;
                     vm.team.error = error.team_name[0];
                     vm.stopLoader();
-                    $rootScope.notify("error", "New team couldn't be created.");
+                    if (vm.team.error == "This field is required.") {
+                        $rootScope.notify("error", "Illegal character in team name.");
```

Comment:
Please also show which character is illegal in the name entered by the participant.

`Illegal character in team name.` -> It is way too generic.